operations/base-resource-defaults.yml

```diff
@@ -5,4 +5,4 @@
       registry_mirror:
         host: ((docker-registry-mirror))
     docker-image:
-      registry_mirror: ((docker-registry-mirror))
+      registry_mirror: https://((docker-registry-mirror))
```

Comment:
The docker-image wants the https:// and we removed it from credhub because of the line above.